bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusSlave.java

```diff
@@ -218,7 +217,7 @@ protected void setRegister(Command command, ModbusBindingConfig config)
         } else if (command instanceof UpDownType) {
             newValue = readCachedRegisterValue(readIndex);
             if (newValue == null) {
-                logger.warn("Not polled value for item {}. Cannot process command {}", config.getItemName(), command);
+                logger.warn("Not polled value. Cannot process command {}", command);

```

Comment:
My 1st approach was w/o the config param in this call to keep the API small. 
Since it is back I can revert this.